bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/builder/ChannelBuilder.java

```diff
@@ -57,7 +57,7 @@ private ChannelBuilder(ChannelUID channelUID, @Nullable String acceptedItemType,
     /**
      * Creates a channel builder for the given channel UID and item type.
      *
-     * @param channelUID channel UID
+     * @param channelUID       channel UID
```

Comment:
Why did you reformat some of the comments this way? I thought we already changed this to our project's style.